scripts/pre_commit_linter.py

```diff
@@ -92,6 +92,14 @@
         'message': 'Please make sure all files only have LF endings (no CRLF).',
         'excluded_files': (),
         'excluded_dirs': ()},
+    '<<<<<<<': {
+        'message': 'Please remove Git conflict markers completely. ',
```

Comment:
Actually, you don't want the developer to just remove these, you want them to resolve the merge properly.

So maybe something like: 'Please fully resolve the existing merge conflict.' perhaps?

Ditto below.

Reply:
I agree. Changed as suggested! Thanks!